go.mod

```diff
@@ -102,15 +102,15 @@ require (
 	go.opentelemetry.io/otel v1.28.0
 	go.uber.org/multierr v1.11.0
 	go.uber.org/zap v1.27.0
-	golang.org/x/crypto v0.25.0
-	golang.org/x/exp v0.0.0-20240719175910-8a7402abbf56
-	golang.org/x/mod v0.19.0
-	golang.org/x/net v0.27.0
-	golang.org/x/sync v0.7.0
-	golang.org/x/term v0.22.0
-	golang.org/x/text v0.16.0
+	golang.org/x/crypto v0.26.0
```

Comment:
are all these expected?

Reply:
This is what showed up after `go get github.com/smartcontractkit/chainlink-common@<hash> && make gomodtidy`, so I assume so 🤷‍♂️

Reply:
Ah, these are from Ryan bumping the deps for the generators (in common)